packages/web-components/src/components/cbp-list/cbp-list.scss

```diff
@@ -0,0 +1,58 @@
+/**
+ * @prop --cbp-list-item-color: var(--cbp-color-text-darkest);
+ * @prop --cbp-list-item-color-dark: var(--cbp-color-text-lightest);
+*/
+
+:root {
+  --cbp-list-item-color: var(--cbp-color-text-darkest);
+  --cbp-list-item-color-dark: var(--cbp-color-text-lightest);
+
+  --cbp-list-item-sublist-padding: 3ch;
+
+  // --cbp-list-item-bullet: '-';
+}
+
+
+[data-cbp-theme=light] cbp-list[context*=dark]:not([context=light-always]),
+[data-cbp-theme=dark] cbp-list:not([context=dark-inverts]):not([context=light-always]) {
+  --cbp-list-item-color: var(--cbp-list-item-color-dark);
+}
+
+cbp-list {
+  line-height: var(--cbp-line-height-sm);
+
+  li {
+    color: var(--cbp-list-item-color);
+    font-size: var(--cbp-font-size-body);
+    font-weight: var(--cbp-font-weight-bold);
+  }
+
+  ul ul,
+  ul ol, 
+  ol ul, 
+  ol ol{
+    padding-inline-start: var(--cbp-list-item-sublist-padding);
+  }
+  
+  // //Unordered nested lists
+  ul li{
+    list-style-type: disc;
+   
+    ul li{ //Unordered list 1st sublist
+      list-style-type: '- ';
+    }
+  }
+  //Ordered nested lists
+    ol ol{
```

Comment:
I think you need to set the default marker for ordered lists because the first one under list item 5 has bullets instead of numbers when the top level list is `ul`.